packages/authentication/docs/authentication-action.md

```diff
@@ -0,0 +1,26 @@
+### Auth action
+
+```ts
+import * as HttpErrors from 'http-errors';
+
+async action(request: Request): Promise<UserProfile | undefined> {
+    const authStrategy = await this.getAuthStrategy();
+    if (!authStrategy) {
```

Comment:
How do we differentiate between the following two cases?

1. No authentication is required
2. No strategy is found to handle the authentication

Reply:
@raymondfeng I would expect the resolver `getAuthStrategy()` to throw an error if the strategy is not found.

We haven't added the resolver, I plan to implement it in the extension story in https://github.com/strongloop/loopback-next/issues/2312